src/dmd/init.d

```diff
@@ -217,76 +217,6 @@ extern (C++) final class ExpInitializer : Initializer
     }
 }
 
-version (all)
-{
-    extern (C++) bool hasNonConstPointers(Expression e)
```

Comment:
this may need to be removed from the headers

Reply:
ok